From mark.a.allyn@intel.com  Wed Feb 17 14:47:31 2010
From: Mark Allyn <mark.a.allyn@intel.com>
Date: Fri,  5 Feb 2010 10:52:46 -0800
Subject: Staging: rar_register: provide better explanation in Kconfig
To: linux-kernel@vger.kernel.org, alan@linux.intel.com, charles.f.johnson@intel.com, greg@kroah.com
Cc: Mark Allyn <mark.a.allyn@intel.com>
Message-ID: <1265395966-16001-1-git-send-email-mark.a.allyn@intel.com>


Provides a better explanation of what this
driver is for in the Kconfig file

Signed-off-by: Mark Allyn <mark.a.allyn@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rar_register/Kconfig |   13 +++++++++++++
 1 file changed, 13 insertions(+)

--- a/drivers/staging/rar_register/Kconfig
+++ b/drivers/staging/rar_register/Kconfig
@@ -14,4 +14,17 @@ config RAR_REGISTER
 		contents of the restricted access region control
 		registers.
 
+		The restricted access region control registers
+		(rar_registers) are used to pass address and
+		locking information on restricted access regions
+		to other drivers that use restricted access regions
+
+		The restricted access regions are regions of memory
+		on the Intel MID Platform that are not accessible to
+		the x86 processor, but are accessible to dedicated
+		processors on board peripheral devices.
+
+		The purpose of the restricted access regions is to
+		protect sensitive data from compromise by unauthorized
+		programs running on the x86 processor.
 endmenu
